data-prepper-plugins/rss-source/src/main/java/org/opensearch/dataprepper/plugins/source/rss/RSSSourceConfig.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.plugins.source.rss;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import jakarta.validation.constraints.NotBlank;
+
+import java.time.Duration;
+
+public class RSSSourceConfig {
+
+    static final Duration DEFAULT_POLLING_FREQUENCY = Duration.ofMinutes(5);
+
+    @JsonProperty("url")
+    @NotBlank(message = "RSS Feed URL cannot be null or empty")
+    private String url;
+
+    @JsonProperty("pollingFrequency")
```

Comment:
What is our expected format for `pollingFrequency`? Can the JsonProperty annotation handle the Duration object conversion?  Can you add a unit test demonstrating this?

Reply:
Added a simple unit test asserting the default here.

Reply:
I am not concerned about the default. I am concerned about the JSON object being parsed into this object. I would like to see a test demonstrating that. I am concerned the current format will not support parsing the config correctly. Feel free to add it in a follow-up PR.